framework/test_knobs.h

```diff
@@ -31,6 +31,8 @@ uint64_t get_testspecific_knob_value_uint(const struct test *test, const char *k
                                           uint64_t value_if_not_present);
 int64_t  get_testspecific_knob_value_int(const struct test *test, const char *key,
                                          int64_t value_if_not_present);
+double   get_testspecific_knob_value_double(const struct test *test, const char *key,
```

Comment:
Can you add the macro in the `RESTRICTED_CMDLINE` section above? (GitHub doesn't let me comment there).

Reply:
Done.